.eslintrc.js

```diff
@@ -172,6 +172,8 @@ module.exports = {
     // ES 2018 - specify migrated files and folders here
     files: [
       'bin/db-maintenance/ensure-indexes.js',
+      'config/env/**',
+      'config/webpack/**',
```

Comment:
These are really backend code, but they were in the clientside eslint list.

Reply:
On second look (when they failed on linters), they're actually kinda mixture. 🤔 

Reverting this change (better anyway for more focused PR).